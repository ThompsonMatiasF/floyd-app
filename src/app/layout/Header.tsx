import React from 'react'
import moment from 'moment'

const Header = () => (
  <div className="flex flex-row justify-between items-end py-4 px-8 md:px-0">
    <span
      className="font-bold text-3xl md:text-5xl"
      style={{fontFamily: 'Berkshire Swash'}}
    >
      <span className="text-red-500">floyd</span> app</span>
    <span className="uppercase font-bold text:xl md:text-2xl">{moment().format('MMMM YYYY')}</span>
  </div>
)

export default Header
