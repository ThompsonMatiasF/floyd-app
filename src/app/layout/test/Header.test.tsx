import { render } from '@testing-library/react'
import Header from '../Header'

describe('Header.tsx', () => {
  it('should match snapshot', () => {
    const container = render(<Header />).container

    expect(container).toMatchSnapshot()
  })
})

