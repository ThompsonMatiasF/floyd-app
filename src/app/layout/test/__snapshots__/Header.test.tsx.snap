// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header.tsx should match snapshot 1`] = `
<div>
  <div
    class="flex flex-row justify-between items-end py-4 px-8 md:px-0"
  >
    <span
      class="font-bold text-3xl md:text-5xl"
      style="font-family: Berkshire Swash;"
    >
      <span
        class="text-red-500"
      >
        floyd
      </span>
       app
    </span>
    <span
      class="uppercase font-bold text:xl md:text-2xl"
    >
      January 2021
    </span>
  </div>
</div>
`;
